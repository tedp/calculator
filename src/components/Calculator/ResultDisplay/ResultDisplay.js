import React from 'react';
import './ResultDisplay.scss';

function ResultDisplay({ displayValue }) {
  return <div className="ResultDisplay">{displayValue}</div>;
}

export default ResultDisplay;
