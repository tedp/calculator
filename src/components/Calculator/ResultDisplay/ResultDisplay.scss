@import '../../../styles/styles.scss';

$module: 'ResultDisplay';

.#{$module} {
  border: 1px solid $dark-grey;
  border-radius: 5px;
  height: 60px;
  padding-right: 15px;
  padding-top: $padding;
  text-align: right;
  margin: 0 10px 5px 5px;
  font-size: 2.5rem;
  overflow-x: auto;
}
  