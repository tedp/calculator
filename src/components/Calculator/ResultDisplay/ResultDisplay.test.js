import React from 'react';
import { shallow } from 'enzyme';
import ResultDisplay from './ResultDisplay';

describe('ResultDisplay', () => {
  it('should render correctly', () => {
    const displayValue = '100';
    const component = shallow(<ResultDisplay displayValue={displayValue} />);

    expect(component).toMatchSnapshot();
  });
});
