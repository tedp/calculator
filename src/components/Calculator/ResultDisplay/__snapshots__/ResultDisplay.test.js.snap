// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultDisplay should render correctly 1`] = `
<div
  className="ResultDisplay"
>
  100
</div>
`;
